docs/contributing/workflow.rst

```diff
@@ -128,10 +128,16 @@ repeated multiple times.
    Committing a change is like preserving a snapshot of what each file
    looks like at this point in history.
 
-   If it has been installed, pre-commit will perform automated checks
-   and possibly make some automated changes. If pre-commit fails, then
-   it'll be necessary to do the ``git add`` and ``git commit`` steps
-   once more.
+
+   .. hint::
+
+      If it has been installed, |pre-commit| will perform automated
+      checks and possibly auto-fixes. If pre-commit fails, then
+      it'll be necessary to fix any remaining problems and do the
+      ``git add`` and ``git commit`` steps once more. Try using
+      ``git diff`` and ``git diff --cached`` to view the changes, and
+      :guilabel:`↑` and :guilabel:`↓` to scroll through previous
+      commands in a terminal.
```

Comment:
```suggestion
      If |pre-commit| has been installed, it will perform automated
      checks and possibly auto-fixes. If pre-commit fails, then
      it'll be necessary to fix any remaining problems and do the
      ``git add`` and ``git commit`` steps once more. 

      Use ``git diff`` and ``git diff --cached`` to view the changes,
      and :guilabel:`↑` and :guilabel:`↓` to scroll through previous
      commands in a terminal.
```